experimenter/tests/integration/nimbus/utils/filter_expression.js

```diff
@@ -1,55 +1,48 @@
-async function remoteSettings(arguments) {
-
+async function remoteSettings(targetingString, recipe) {
     /*
-        Arguments contains 2 items.
-        arguments[0] - the JEXL targeting string
-        arguments[1] - the experiment recipe
+    Arguments contains 2 items.
+    arguments[0] - the JEXL targeting string
+    arguments[1] - the experiment recipe
     */
```

Comment:
nit i would put this above the arguments destructuring at the end of the file. Also update for the callback param